osu.Game.Rulesets.Taiko/UI/DrumTouchInputArea.cs

```diff
@@ -163,6 +218,19 @@ protected override void PopOut()
             mainContent.FadeOut(300);
         }
 
+        private Color4 getColourFromTaikoAction(TaikoAction handledAction)
+        {
+            #pragma warning disable format
+            switch (handledAction)
+            {
+                case TaikoAction.LeftRim:     return colours.Blue;
+                case TaikoAction.LeftCentre:  return colours.Red;
+                case TaikoAction.RightCentre: return colours.Red;
+                case TaikoAction.RightRim:    return colours.Blue;
+            }
```

Comment:
More readable as this
```suggestion
            switch (handledAction)
            {
                case TaikoAction.LeftRim:
                case TaikoAction.RightRim:
                    return colours.Blue;
                case TaikoAction.LeftCentre:
                case TaikoAction.RightCentre:
                    return colours.Red;
            }
```